src/wallet/rpcwallet.cpp

```diff
@@ -2758,10 +2758,10 @@ UniValue settxfee(const JSONRPCRequest& request)
         return NullUniValue;
     }
 
-    if (request.fHelp || request.params.size() < 1 || request.params.size() > 1)
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 1) {
         throw std::runtime_error(
             "settxfee amount\n"
-            "\nSet the transaction fee per kB. Overwrites the paytxfee parameter.\n"
+            "\nSet the transaction fee per kB for this wallet. Overwrites the global -paytxfee command line parameter.\n"
```

Comment:
Comment is kind of misleading. I think it would be more accurate to say "overrides" instead of "overwrites" now.

Reply:
TIL those words mean different things. Thx and done. Should be trivial to re-ACK if you still have the previous commit.